### PR TITLE
Merged GA into -cp branch after releasing multiple CVEs

### PR DESCRIPTION
**Pull request by osnipso**
Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [X] Pull Request contains description of the issue
- [X] Pull Request does not include fixes for other issues
- [X] Attached commits represent units of work and are properly formatted
- [X] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [X] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: image.yaml [hunk 1]
```diff
@@ -53,5 +55,5 @@ run:
           - "0.0.0.0"
 osbs:
       repository:
-            name: jboss-datagrid-7-docker
-            branch: jb-datagrid71
+            name: containers/jboss-datagrid-7
+            branch: jb-datagrid-7.1-rhel-7
```
Comment (rcernich) [inline image.yaml:59]: Is this correct?

**Review by rcernich — APPROVED** [review]

File: image.yaml (hunk 1 above)
Comment (osnipso) [reply image.yaml:59]: Yes, we have switched to the new containers namespace for the standalone image in the last CVE re-spin, following Marek's advice, so all branches (ga, -dev and -cp) should use it. Also, when building from -dev or -cp, I'm using an overrides.yaml file (to select the proper -dev or -cp branch), I shall open PRs to add them to our repo next.

Comment (osnipso) [reply image.yaml]: Here's the reasoning: https://docs.engineering.redhat.com/display/CE/2018/03/14/Container+namespace+and+standalone+images

Comment (rcernich) [reply image.yaml:59]: Thanks, just verifying.

**Pull request merged**